src/components/YourCompany/NameTranslations/NameTranslation.vue

```diff
@@ -0,0 +1,314 @@
+<template>
+  <div id="name-translation">
+    <confirm-dialog
+      ref="confirmTranslationDialog"
+      attach="#name-translation"
+    />
+    <v-layout class="mt-3" v-if="!isEditing">
+      <v-flex md3>
+        <label><strong>Name Translation(s)</strong></label>
+        <v-flex md1>
+            <v-chip v-if="hasNameTranslationChange" x-small label color="#1669BB" text-color="white">
+                Corrected
+            </v-chip>
+          </v-flex>
+      </v-flex>
+      <v-flex md7 v-if="drafTranslations && drafTranslations.length">
+        <div v-for="(name, index) in drafTranslations" :key="`name_translation_${index}`">{{name}}</div>
+      </v-flex>
+      <v-flex md7 v-else>
+        No name translations
+      </v-flex>
+      <v-flex md2 class="align-right" v-if="!hasNameTranslationChange">
+        <v-btn
+          text color="primary"
+          @click="isEditing = true"
+        >
```

Comment:
I recommend adding an id to buttons that appear only once, or a class to buttons that can appear multiple times, for testing purposes!

Reply:
OK, thanks.

Please also update the buttons on lines 105 and 111.